src/Spago/Messages.hs

```diff
@@ -146,6 +146,18 @@ upgradingPackageSet newTag = makeMessage
   , "Fetching the new one and generating hashes.. (this might take some time)"
   ]
 
+changingToSpecificPackageSet :: Text -> Text
+changingToSpecificPackageSet newTag = makeMessage
+  [ "Current tag is different from specified tag " <> surroundQuote newTag
```

Comment:
I think we can unify this with the previous message:
```suggestion
  [ "Updating package-sets tag to: " <> surroundQuote newTag
```

Reply:
Done!